docs/md/modules-arrays.md

```diff
@@ -80,47 +80,69 @@ To give more information, here are some example inputs to the above endpoints:
     hitting this endpoint without them will return all workloads. However, they cannot be specified
     together.
 
-**GET /api/v1/workload/create**
+**GET /api/v1/workload/{uuid}/workflows**
 
 === "Request"
 
     ```shell
-    curl -X "POST" "http://localhost:8080/api/v1/create" \
-        -H 'Authorization: Bearer '$(gcloud auth print-access-token) \
-        -H 'Accept: application/json' \
-        -H 'Content-Type: application/json' \
-        -d '{
-                "executor": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
-                "output":   "gs://broad-gotc-dev-wfl-ptc-test-outputs/aou-test-output/",
-                "project":  "Example Project",
-                "pipeline": "AllOfUsArrays"
-            }'
+    curl 'http://localhost:8080/api/v1/workload/00000000-0000-0000-0000-000000000000/workflows' \
+        -H 'Authorization: Bearer '$(gcloud auth print-access-token) 
     ```
 
-**GET /api/v1/workload/start**
+**POST /api/v1/workload/create**
 
 === "Request"
 
     ```shell
-    curl -X "POST" "http://localhost:8080/api/v1/start" \
-        -H 'Authorization: Bearer '$(gcloud auth print-access-token) \
-        -H 'Accept: application/json' \
-        -H 'Content-Type: application/json' \
-        -d $'{
-                "uuid": "00000000-0000-0000-0000-000000000000"
-              }'
+    curl -X POST 'http://localhost:8080/api/v1/create' \
+         -H 'Authorization: Bearer '$(gcloud auth print-access-token) \
+         -H 'Accept: application/json' \
+         -H 'Content-Type: application/json' \
+         -d '{
+               "executor": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
+               "output":   "gs://broad-gotc-dev-wfl-ptc-test-outputs/aou-test-output/",
+               "project":  "Example Project",
+               "pipeline": "AllOfUsArrays"
+             }'
     ```
 
-**GET /api/v1/workload/append_to_aou**
+**POST /api/v1/workload/start**
 
 === "Request"
 
     ```shell
-    curl -X "POST" "http://localhost:8080/api/v1/append_to_aou" \
-        -H 'Authorization: Bearer '$(gcloud auth print-access-token) \
-        -H 'Accept: application/json' \
-        -H 'Content-Type: application/json' \
-        -d $'{
+    curl -X POST 'http://localhost:8080/api/v1/start' \
+         -H 'Authorization: Bearer '$(gcloud auth print-access-token) \
+         -H 'Accept: application/json' \
+         -H 'Content-Type: application/json' \
+         -d $'{ "uuid": "00000000-0000-0000-0000-000000000000" }'
+    ```
+
+**POST /api/v1/workload/stop**
+
+Stops the workload from accepting new inputs.
+See also: `/api/v1/append_to_aou`.
```

Comment:
Is this line meant to fall in this section?

Reply:
What do you mean? Are you asking if a copypasta?

Reply:
Yeah, it didn't make sense to me that `append_to_aou` would be related to stopping a workload.

Reply:
Ah, because this workload can be stopped!